### ISSUE TITLE
request: adding effects with plus

### ISSUE THREAD

**Issue by M0ahgra** [johnnesky/beepbox#96]
add things with the plus symbol, remove them with the x symbol.
subinstruments work this way,
envelopes work this way,
but effects dont work this way.

i believe it could b easier to work with effects if they worked in a similar way to the above.

**Comment by Mid-The-Modder** [comment]
Agreed. Having little subsections in the UI where you could collapse/remove effects would be really nice.

**Comment by M0ahgra** [comment]
such a thing could allow us to have multiple of the same effect in an instrument, and with the addition of the ability to reorder them, it would be a real treat.
